packages/gasket-cli/lib/utils/warn-if-outdated.js

```diff
@@ -0,0 +1,75 @@
+/* eslint-disable no-sync */
+const { runShellCommand } = require('@gasket/utils');
+const semver = require('semver');
+const chalk = require('chalk');
+const fs = require('fs');
```

Comment:
```suggestion
const { access, readFile, writeFile } = require('fs/promises');
```
You already have an `async` function so why not 🤷‍♂️